test/__mocks__/data/salesforce/proxy_response.json

```diff
@@ -1,4 +1,44 @@
 {
+  "https://rudderstack.my.salesforce.com/services/data/v50.0/sobjects/Lead/101": {
+    "message": "timeout of 1ms exceeded",
+    "name": "Error",
+    "stack": "Error: timeout of 1ms exceeded\n at createError (webpack-internal:///./node_modules/axios/lib/core/createError.js:16:15)\n at XMLHttpRequest.handleTimeout (webpack-internal:///./node_modules/axios/lib/adapters/xhr.js:89:14)",
+    "config": {
+      "url": "redacted",
+      "method": "get",
+      "headers": {
+        "Accept": "application/json, text/plain, */*"
+      },
+      "transformRequest": [
+        null
+      ],
+      "transformResponse": [
+        null
+      ],
+      "timeout": 1
+    },
```

Comment:
Can you exclude this part as I don't think these are the correct values ? 
```suggestion
  
```
Or you can put some message which clearly states relevance of the code(`ECONNABORTED` or `EAI_AGAIN`)

Reply:
Removed the extra things just the message and code is there